experimental/Schemes/SpaceGerms.jl

```diff
@@ -0,0 +1,274 @@
+export SpaceGerm, germ_at_point
+
+export representative, point
+
+export ambient_germ
+
+import AbstractAlgebra: Ring
+
+@Markdown.doc """
+    AbsSpaceGerm{BaseRingType<:Ring,RingType<:Ring}
+
+A space germ, i.e. a ringed space ``(X,O_{(X,x)})`` with local ring ``O_{(X,x)}`` of type `RingType` over a ring ``k`` of type `BaseRingType`.
```

Comment:
In the above text you emphasize the *field* of coefficients. Here we find a base*ring*. Should this be streamlined?

Reply:
yes, done.